File: DOLL.py
# cook your dish here
for i in range(int(input())):
    count=0
    n=input()
    k=input()
    n=n.split()
    k=k.split()
    for j in k:
        if int(j)>int(n[1]):
            count+=1
    print(count)

File: DOLL.cpp
#include <iostream>
using namespace std;

int main() {
	// your code goes here
	int t;
	cin>>t;
	while(t--){
	    int n,k;
	    cin>>n>>k;
	    int count=0;
	    for(int i=0;i<n;i++){
	        int temp;
	        cin>>temp;
	        if(temp>k){
	            count++;
	        }
	    }
	    cout<<count<<endl;
	}
	return 0;
}
